src/app/_components/DemoHero/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './DemoHero';
```

Comment:
👍

Reply:
Sure, I can remove it unless other guys need a reference component to look at.